cockatrice/src/userinfobox.cpp

```diff
@@ -199,7 +201,22 @@ void UserInfoBox::actEditInternal(const Response &r)
 
     Command_AccountEdit cmd;
     cmd.set_real_name(dlg.getRealName().toStdString());
-    cmd.set_email(dlg.getEmail().toStdString());
+    if (client->getServerSupportsPasswordHash()) {
```

Comment:
A client can be modified to remove this check -- Haven't gotten there, but does the server reject the change if the passwordCheck isn't set when the server users hashed passwords and the user fails to provide the plaintext one?

Reply:
of course, it returns RespFunctionNotAllowed https://github.com/Cockatrice/Cockatrice/pull/4493/commits/5129f40461e10519e758c8b65e6927327a7f78c1#diff-16bda30986b55ad12ce94d6bd6d89fce0f84793010faa23a893d8194dbf84530R1294